src/services/access.service.ts

```diff
@@ -9,11 +9,22 @@ const AccessService = {
   getAccountsForOrg,
   getDetailedAccountsForOrg,
   getIndividualAccount,
+  isSynapseEmail,
   updateIndividualAccountRoles,
 }
 
-async function getAliasFromSynapseByEmail(synapseEmailAddress: string): Promise<any> {
-  if (synapseEmailAddress.indexOf('@synapse.org') === -1 && synapseEmailAddress.indexOf('@synapse.org') > -1) {
+function isSynapseEmail(emailAddress: string): boolean {
+  return emailAddress.replace(new RegExp('.*@'), '') === 'synapse.org'
+}
+
+export type SynapseAlias = {
+  principalId: string
+  firstName: string
+  lastName: string
+}
+
+async function getAliasFromSynapseByEmail(synapseEmailAddress: string): Promise<SynapseAlias> {
+  if (!isSynapseEmail(synapseEmailAddress)) {
     return Promise.reject({message: 'the email should be ...@synapse.org'})
   }
```

Comment:
This doesn't need to be asynchronous, suggest throwing an error instead of rejecting a promise.